### DISCUSSION TITLE
Logfile to report file data tables

### DISCUSSION THREAD

**Discussion by LuiseMW** [davidfrantz/force#291] [Q&A] [answered]
Is the are an option to access the data that is used to generate the reports of Level 2 and 3 (Landsat 8 and Sentinel-2) in table format e.g.  .csv?
We would like to further analyse the results of the product and have more information about the process.

Thanks!

**Comment by ernstste** [comment]
The reports are generated by parsing the Level 2 log files. I don't think that any intermediate csv tables are being generated, but you can check the R code [here](https://github.com/davidfrantz/force/blob/main/rstats/force-level2-report.Rmd) and adapt it to provide the data you need.

**Reply by davidfrantz** [reply]
That's correct, no table is generated. However, the code compiles a data frame (ll. 104-141). If you add a ``write.csv()`` function, you should have what you need.

Cheers,
David

**Comment by davidfrantz** [comment, marked as answer]
This is now implemented per default in dev
